src/Notes/WC_Admin_Notes_Performance_On_Mobile.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WooCommerce Admin Performance on mobile note.
+ *
+ * Adds a note to download the mobile app, performance on mobile.
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Performance_On_Mobile
+ */
+class WC_Admin_Notes_Performance_On_Mobile {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-performance-on-mobile';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+		// Only add this note if this store is at least 9 months old.
+		$nine_months_in_seconds = 365 / 12 * 9 * DAY_IN_SECONDS;
```

Comment:
Just in case you were unaware, there is a `MONTH_IN_SECONDS` constant as well, but it's a slightly different approximate value: 30 days.

Reply:
oh no. Thanks, I'll use that constant and try not to get too triggered by the inaccuracy 😱